### PR TITLE
Sql and work shift

### PR DESCRIPTION
**Pull request by warfawl**

### CODE REVIEW

File: src/main/java/ua/danit/final_project/entities/Schedule.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @ManyToOne(fetch = FetchType.LAZY, optional = false)
+  @JoinColumn(name = "r_id", nullable = false)
+  private Role role;
+
+  private Timestamp start;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Schedule.java]: Code Smell Code Smell: Remove this unused "start" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-hW2AOP0Ck_lbR&open=AWUy2H-hW2AOP0Ck_lbR)

[ISSUE_KEY]: AWUy2H-hW2AOP0Ck_lbR

File: src/main/java/ua/danit/final_project/entities/Schedule.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  private Timestamp end;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Schedule.java]: Code Smell Code Smell: Remove this unused "end" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-hW2AOP0Ck_lbS&open=AWUy2H-hW2AOP0Ck_lbS)

[ISSUE_KEY]: AWUy2H-hW2AOP0Ck_lbS

File: src/main/java/ua/danit/final_project/entities/WorkShift.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+  @ManyToOne
+  @JoinColumn(name = "u_id", nullable = false)
+  private User user;
+
+  @ManyToOne
+  @JoinColumn(name = "s_id", nullable = false)
+  private Schedule schedule;
+
+  private Timestamp date;
```
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/WorkShift.java]: Code Smell Code Smell: Remove this unused "date" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-mW2AOP0Ck_lbT&open=AWUy2H-mW2AOP0Ck_lbT)

[ISSUE_KEY]: AWUy2H-mW2AOP0Ck_lbT

**Comment by warfawl** [comment]
[Создать сущности для смен/расписания](https://trello.com/c/XmbPKt85/117-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D1%82%D1%8C-%D1%81%D1%83%D1%89%D0%BD%D0%BE%D1%81%D1%82%D0%B8-%D0%B4%D0%BB%D1%8F-%D1%81%D0%BC%D0%B5%D0%BD-%D1%80%D0%B0%D1%81%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D1%8F)

**Comment by warfawl** [comment]
[Создать сущности для смен/расписания](https://trello.com/c/XmbPKt85/117-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D1%82%D1%8C-%D1%81%D1%83%D1%89%D0%BD%D0%BE%D1%81%D1%82%D0%B8-%D0%B4%D0%BB%D1%8F-%D1%81%D0%BC%D0%B5%D0%BD-%D1%80%D0%B0%D1%81%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D1%8F)

**Comment by warfawl** [comment]
[Создать сущности для смен/расписания](https://trello.com/c/XmbPKt85/117-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D1%82%D1%8C-%D1%81%D1%83%D1%89%D0%BD%D0%BE%D1%81%D1%82%D0%B8-%D0%B4%D0%BB%D1%8F-%D1%81%D0%BC%D0%B5%D0%BD-%D1%80%D0%B0%D1%81%D0%BF%D0%B8%D1%81%D0%B0%D0%BD%D0%B8%D1%8F)

File: src/main/java/ua/danit/final_project/entities/Schedule.java (hunk 2 above)
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Schedule.java]: Code Smell Code Smell: Remove this unused "end" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-hW2AOP0Ck_lbS&open=AWUy2H-hW2AOP0Ck_lbS)

[ISSUE_KEY]: AWUy2H-hW2AOP0Ck_lbS

File: src/main/java/ua/danit/final_project/entities/WorkShift.java (hunk 3 above)
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/WorkShift.java]: Code Smell Code Smell: Remove this unused "date" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-mW2AOP0Ck_lbT&open=AWUy2H-mW2AOP0Ck_lbT)

[ISSUE_KEY]: AWUy2H-mW2AOP0Ck_lbT

File: src/main/java/ua/danit/final_project/entities/Schedule.java (hunk 1 above)
Comment (crystweb) [inline src/main/java/ua/danit/final_project/entities/Schedule.java]: Code Smell Code Smell: Remove this unused "start" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-hW2AOP0Ck_lbR&open=AWUy2H-hW2AOP0Ck_lbR)

[ISSUE_KEY]: AWUy2H-hW2AOP0Ck_lbR

**Pull request merged**